### PR TITLE
Release 2.0.1

### PR DESCRIPTION
**Pull request by PierreRambaud**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Release 2.0.1.
| Type?         | improvement

### CODE REVIEW

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+GitHub contributors:
+--------------------------------
+ - Francois Gaillard
+ - Jerome Nadaud
```
Comment (atomiix) [inline CONTRIBUTORS.md]:
```suggestion
```

File: upgrade/upgrade-2.0.1.php [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/AFL-3.0 Academic Free License 3.0 (AFL-3.0)
+ */
+
+/**
+ * @param Module $module
+ *
+ * @return bool
+ */
+function upgrade_module_2_0_1$module)
```
Comment (PierreRambaud) [inline upgrade/upgrade-2.0.1.php]:
```suggestion
function upgrade_module_2_0_1($module)
```

**Comment by florine2623** [comment]
Hello @PierreRambaud ,

Thanks for the PR, it is QA ✅ !

**Review by Progi1984 — APPROVED** [review]

**Comment by Progi1984** [comment]
Thanks @PierreRambaud & @florine2623

**Pull request merged**